client/src/components/notifications/NewDeviceNotification.vue

```diff
@@ -0,0 +1,63 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <notification-item
+    :notification="notification"
+    @click="navigateToDevices"
+  >
+    <div class="notification-icon">
+      <ion-icon
+        class="file-icon"
+        :icon="desktop"
+      />
+    </div>
+    <div class="notification-details">
+      <ion-text class="notification-details__message body">
+        {{ $t('notification.newDeviceAdded') }}
+      </ion-text>
+      <ion-text class="notification-details__time body-sm">
+        <span class="hover-state">
+          {{ $t('notificationCenter.goToDevices') }}
+        </span>
+        <span class="default-state">{{ formatTimeSince(notification.time, '', 'short') }}</span>
+        <ion-icon
+          class="arrow-icon hover-state"
+          :icon="arrowForward"
+        />
+      </ion-text>
+    </div>
+  </notification-item>
+</template>
+
+<script setup lang="ts">
+import { formatTimeSince } from '@/common/date';
+import NotificationItem from '@/components/notifications/NotificationItem.vue';
+import { Routes, navigateTo } from '@/router';
+import { Notification } from '@/services/notificationManager';
+import { IonIcon, IonText, popoverController } from '@ionic/vue';
+import { arrowForward, desktop } from 'ionicons/icons';
+
+defineProps<{
+  notification: Notification;
+}>();
+
+async function navigateToDevices(): Promise<void> {
+  await navigateTo(Routes.Devices);
```

Comment:
Dismiss the popover before navigating to the new route.

Reply:
I swapped them.